app/constants/service-name.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  serviceName: 'aphw-ddi-portal'
```

Comment:
This will need changing to 'aphw-ddi-enforcement'

Reply:
good catch